examples/volume.go

```diff
@@ -50,6 +50,21 @@ func main() {
 		fmt.Printf("Failed to create volume, err ", err)
 		return
 	}
+
+	// get volume with params
+	requestParams := new(param.GetParams)
+	fieldList := []string{
+		*nimbleos.VolumeFields.ID,
+		*nimbleos.VolumeFields.Name,
+		*nimbleos.VolumeFields.Size,
+		*nimbleos.VolumeFields.LimitMbps,
+	}
+	requestParams.WithFields(fieldList)
+	_, err  = volSvc.GetVolumes(requestParams)
```

Comment:
Should we print the result or something?  Maybe just the first volume in the list.